x-pack/test/functional/apps/ml/anomaly_detection/single_metric_job_without_datafeed_start.ts

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const ml = getService('ml');
+
+  const jobId = `fq_single_1_${Date.now()}`;
+  const aggAndFieldIdentifier = 'Mean(responsetime)';
+  const bucketSpan = '30m';
+
+  function getExpectedRow(expectedJobId: string) {
+    return {
+      id: expectedJobId,
+      description: '',
+      jobGroups: [],
+      recordCount: '0',
+      memoryStatus: 'ok',
+      jobState: 'closed',
+      datafeedState: 'stopped',
+      latestTimestamp: '',
+    };
+  }
+
+  function getExpectedCounts(expectedJobId: string) {
+    return {
+      job_id: expectedJobId,
+      processed_record_count: '0',
+      processed_field_count: '0',
+      input_bytes: '0.0 B',
+      input_field_count: '0',
+      invalid_date_count: '0',
+      missing_field_count: '0',
+      out_of_order_timestamp_count: '0',
+      empty_bucket_count: '0',
+      sparse_bucket_count: '0',
+      bucket_count: '0',
+    };
+  }
+
+  function getExpectedModelSizeStats(expectedJobId: string) {
+    return {
+      job_id: expectedJobId,
+      result_type: 'model_size_stats',
+      total_by_field_count: '0',
+      total_over_field_count: '0',
+      total_partition_field_count: '0',
+      bucket_allocation_failures_count: '0',
+      memory_status: 'ok',
+    };
+  }
+
+  const calendarId = `wizard-test-calendar_${Date.now()}`;
+
+  describe('single metric without datafeed start', function () {
+    this.tags(['mlqa']);
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/farequote');
+      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+
+      await ml.api.createCalendar(calendarId);
```

Comment:
The calendar is not used in this test, so this could be removed together with the `calendarId` definition above.

Reply:
Updated in https://github.com/elastic/kibana/pull/77484/commits/252c1c7d5717f99efd4ce8c093e589d4fc6c0dbb